src/components/SwatchPaletteWidget/SwatchPalette.js

```diff
@@ -1,554 +1,570 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { settings } from 'carbon-components';
+import {
+  black,
+  blue,
+  coolGray,
+  warmGray,
+  white,
+  yellow,
+  magenta,
+  orange,
+  teal,
+  purple,
+  red,
+  green,
+  gray,
+  cyan,
+} from '@carbon/colors';
 
 const { prefix } = settings;
 
 const PALETTES = {
   black: {
     '100': {
-      hex: '#000000',
+      hex: black,
       pms: 'Black 6',
-      cmyk: 'c75 m68 y67 k90',
+      cmyk: 'c60 m40 y40 k100',
```

Comment:
Reverted the CMYK colors according to the pdf values. The PDF has a different value for black than it did before. The rest of the colors remain the same, however.
<img width="316" alt="Screen Shot 2019-11-05 at 3 18 41 PM" src="https://user-images.githubusercontent.com/32556167/68247105-bc182680-ffdf-11e9-9fa5-5c348e3af852.png">